Catappult/appcoins-wallet-android
# [APPC-3496] FAQs in the settings

**What does this PR do?**

 Add FAQs to settings

**Database changed?**

No

**Where should the reviewer start?**


**How should this be manually tested?**

Go to settings and click in FAQs option and check if you get redirected to wallet.appcoins.io/faqs and that the sent language is the correct one.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-3496](https://aptoide.atlassian.net/browse/APPC-3496)

**Code Review Checklist**
